src/main/scala/com/gu/newsletterlistcleanse/db/BigQueryOperations.scala

```diff
@@ -37,7 +37,7 @@ class BigQueryOperations(serviceAccount: String, projectId: String) extends Data
   override def fetchCampaignSentDates(campaignNames: List[String], cutOffLength: Int): List[CampaignSentDate] = {
 
     val sql = """SELECT campaign_name, campaign_id, timestamp FROM (
-                |  SELECT row_number() over(PARTITION BY campaign_name) AS rn, *
+                |  SELECT row_number() over(Partition by campaign_id ORDER BY campaign_id, timestamp desc) AS rn, *
```

Comment:
```suggestion
                |  SELECT row_number() over(PARTITION BY campaign_id ORDER BY campaign_id, timestamp desc) AS rn, *
```

Capitalisation to make the SQL clause more visible as compared to the column names

Reply:
Thanks!